src/components/panel/panel.yaml

```diff
@@ -8,3 +8,4 @@ variants:
     classes: extra-dummy-class
     titleText: Application complete
     text: Thank you for your application
+    
```

Comment:
I think your second commit is just removing the new line from the end of the file – which we don't want. Can we remove it?